### DISCUSSION TITLE
Houseware Octernship questions:

### DISCUSSION THREAD

**Discussion by ishaan812** [community/community#49953] [GitHub Education]
Hey, love the new octernships to reward people actively contributing on Github. I had some questions about one of the tasks about building an Authentication service in Go from Housewares. 
Did we have to stick to the structure that has been given to us for the Go Project? 
It felt quite excessive to me.

**Comment by samus-aran** [comment]
Hi @ishaan812 

You can reach out to the team behind the project by creating an issue on the repo designated for the Octernship. I'm sure the team will provide guidance if issues are the best place once you create one

**Comment by shikharcodess** [comment]
It is clearly mentioned in the `README.md` file in `/backend` folder that, "not all is for you, feel free to modify/add/delete any of the file and folder.
